src/devices/Adxl345/README.md

```diff
@@ -0,0 +1,28 @@
+# ADXL345
+ADXL345 is a small, thin, low power, 3-axis accelerometer with high resolution (13-bit) measurement at up to ±16g.
+
+## Sensor Image
+![](sensor.jpg)
+
+## Usage
+* First, you need to create a ADXL345 object. After that you should call Initialize() to initialize.
+    ```C#
+    // OSPlatform, SPI Bus ID, CS Pin, GravityRange
+    Adxl345 sensor = new Adxl345(OSPlatform.Linux, 0, 0, GravityRange.Four);
```

Comment:
Perhaps the sample should be something like

```csharp
using (Adxl345 sensor = new Adxl345(new SpiDevice(0, 0), GravityRange.Four))
{
  Console.WriteLine(sensor.Acceleration);
}
```

most of the comments are not neccesary, perhaps just say "Example". Code is fairly self-explanatory